tests/news/san_mateo_test.py

```diff
@@ -0,0 +1,109 @@
+from covid19_sfbayarea.news.san_mateo import SanMateoNews
```

Comment:
Hey this is awesome! Great example of how to unit test something that I thought was near impossible to unit test.